src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -59,6 +61,11 @@ public class Drivetrain extends SubsystemBase {
     // If the robot should aim at the speaker
     private boolean isAlign = false;
 
+    // If the shoot command is currently running, and the swerve angular speed should be locked
+    private boolean isShooting = false;
+    // Radians in which the shoot command is requesting
+    public double shootAngle = 0;
```

Comment:
These are unnecessary. Use isAlign instead. The calculations should either be in Shooter or Drivetrain, not in any specific command. If you want them to be in Shooter, then you can add a variable, alignAngle or shootAngle, that is returned in getAlignAngle() instead of the angle it currently calculates.